dcr/scenarios/agent-bvt/run2.py

```diff
@@ -3,16 +3,13 @@
 from dcr.scenario_utils.check_waagent_log import check_waagent_log_for_errors
 from dcr.scenario_utils.models import get_vm_data_from_env
 from dcr.scenario_utils.test_orchestrator import TestFuncObj, TestOrchestrator
-from get_blob_content import show_blob_content
 from test_agent_basics import check_agent_processes, check_sudoers
 
 if __name__ == '__main__':
     admin_username = get_vm_data_from_env().admin_username
     tests = [
         TestFuncObj("check agent processes", check_agent_processes),
         TestFuncObj("check agent log", check_waagent_log_for_errors),
-        TestFuncObj("Verify status blob", lambda: show_blob_content('Status', 'StatusUploadBlob')),
```

Comment:
These tests are just parsing /var/lib/waagent/ExtensionsConfig.*.xml and dumping the status and artifacts profile blob. apparently for debugging purposes. 

That file does not exist anymore, but the equivalent is in the history folder, which is already captured by automation.

Once we enable Fast Track this "test" would need significant changes to make it aware of vmsettings. Since we already have the debug info in the history, it is not worth to update it.